docs/api/policy.rst

```diff
@@ -0,0 +1,15 @@
+.. _policy:
+
+Tax-Calculator Policy
+=================================================
+
+**Tax-Calculator Policy**
+
+taxcalc.policy
+------------------------------------------
+
+.. currentmodule:: taxcalc.policy
+
+.. autoclass:: Policy
+  :members: read_json_reform, implement_reform, parameter_list,
+    set_rates
```

Comment:
@jdebacker I think you can just add the method names here and jupyter book will pull them off the `Policy` class. When I did this locally, I lost the link to the source code, but that's probably ok for these methods.